src/pytest_postgresql/executor_noop.py

```diff
@@ -67,7 +67,7 @@ def version(self) -> Any:
             # could be called before self.dbname will be created.
             # Use default postgres database
             with psycopg.connect(
-                dbname="postgres",
+                dbname=self.dbname or "postgres",
```

Comment:
but dbname will never be empty 🤔 And we'll attempt to create the database in question.... most probably prior to it's creation 🤔

Reply:
Hmmm. Indeed, `_noproc` tries to create a database, and fails. I'd like to be able to use it with a basic database account, but the problem goes deeper than just connecting to the `postgres` database :-/

Reply:
With test file `test.py`:
```python
import pytest
from pytest_postgresql import factories

def test_hello(postgresql_noproc):
    assert True
```

Run test against an existing database:
```sh
# create a basic user
createuser -W pytest  # enter password: pytest
# with a database
createdb -O pytest pytest
# run test
pytest --postgresql-user=pytest --postgresql-password=pytest --postgresql-dbname=pytest --postgresql-use-database test.py
```

Reply:
This should be dependant on the new flag you've added. rather than the self.dbname being null which it will not 🤔

Reply:
I'm not sure. This means that the option has somehow two effects, but why not.

I've updated the patch.